src/components/Page/Page.jsx

```diff
@@ -122,10 +122,11 @@ export class Page extends Component {
           _onRedo={() => this.state.boardState.drawBoard.redo()}
           onBrushSizeChange={(size) => this.state.boardState.drawBoard.changeBrushSize(size)}
           onEraserSizeChange={(size) => this.state.boardState.drawBoard.changeEraserSize(size)}
+          _changeToolSetting={(property,newValue) => this.state.boardState.drawBoard.changeToolSetting(property,newValue)}
         />
       </div>
     )
   }
 }
 
-export default Page;
+export default Page;
```

Comment:
Please add the final newline just because it is followed everywhere :stuck_out_tongue:

Reply:
You mean an empty line after this?

Reply:
Yes

Reply:
Ok thanks wil do this now